### PR TITLE
Adding research page

### PR DESCRIPTION
**Pull request by crazy4pi314**

### CODE REVIEW

**Comment by willzeng** [comment]
Looks like a good start!

File: research.html [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                                        href="https://en.wikipedia.org/wiki/Unitarity_(physics)"
+                                        target="_blank">unitary</a></b>.</p>
+                            <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                                <span class="hamburger-box">
+                                    <span class="hamburger-inner"></span>
+                                </span>
+                            </button>
+                        </div>
+                        <p class="subtitle leading-arrow"><a name="research">Research with Unitary Fund</a></p> The Unitary Fund believes that research is an important driver in quantum open source. We focus not only on our own research interests, but also building tools to enable the
+                        research of others and supporting that research via our microgrant program. Check out some of the many research projects we've been involved in or supported below!
```
Comment (willzeng) [inline research.html:91]:
```suggestion
                        <p class="subtitle leading-arrow"><a name="research">Research with Unitary Fund</a></p> Unitary Fund believes that research is important for quantum open source. We focus not only on our own research interests, but also building tools to enable others, like those in our microgrant program. Check out some of the many research projects we've been involved in or supported below!
```

**Comment by crazy4pi314** [comment]
Yeah I have added what was listed above, but maybe we should ask other UF team members if they have other additions?

File: research.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2009.04417">Mitiq: A software package for error mitigation on noisy quantum computers</a>, <a href="https://mitiq.readthedocs.io/en/latest/examples/mitiq-paper/mitiq-paper-codeblocks.html">source code</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2005.10921">Digital zero noise extrapolation for quantum error mitigation</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2108.02237">Extending quantum probabilistic error cancellation by noise scaling</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2105.09902">Pulse-level noisy quantum circuits with QuTiP</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.15044">Pulser: An open-source package for the design of pulse sequences in programmable neutral-atom arrays
+                                </a></li><br>
+                            </ul>
+                        </p>
```
Comment (andreamari) [inline research.html:102]:
```suggestion
                        <p class="subtitle leading-arrow"><a name="research-uf">Unitary Labs research and collaborations</a></p>
                        <p>
                            <ul style="list-style-type: none;">
                                <li class="leading-block"><a href="https://arxiv.org/abs/2009.04417">Mitiq: A software package for error mitigation on noisy quantum computers</a>, <a href="https://mitiq.readthedocs.io/en/latest/examples/mitiq-paper/mitiq-paper-codeblocks.html">source code</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2005.10921">Digital zero noise extrapolation for quantum error mitigation</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2108.02237">Extending quantum probabilistic error cancellation by noise scaling</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2105.09902">Pulse-level noisy quantum circuits with QuTiP</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.15044">Pulser: An open-source package for the design of pulse sequences in programmable neutral-atom arrays
                                </a></li><br>
                            </ul>
                        The Unitary Fund research <a href="https://github.com/unitaryfund/research">GitHub repo</a> has source code for a variety of our research projects.
                        </p>
```

Please ignore this suggestion if don't like it. When reading this part it sounded like we had source code for all the papers listed after the first sentence. So the suggestion would be to to mention the research GitHub repo after the list of papers.

File: research.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                        <p class="subtitle leading-arrow"><a name="research-staff">Unitary Fund staff research</a></p>
+                        <p>
+                            <ul style="list-style-type: none;">
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2101.05710">Symmetries and conserved quantities of boundary time crystals in generalized spin models</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.09811">Exceptional Point and Cross-Relaxation Effect in a Hybrid Quantum System</a></li><br>
+                            </ul>
```
Comment (andreamari) [inline research.html:108]:
```suggestion
                            <ul style="list-style-type: none;">
                                <li class="leading-block"><a href="https://arxiv.org/abs/2101.05710">Symmetries and conserved quantities of boundary time crystals in generalized spin models</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.09811">Exceptional Point and Cross-Relaxation Effect in a Hybrid Quantum System</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2008.06517">Estimating the gradient and higher-order derivatives on quantum hardware</a></li><br>
                            </ul>
```

**Review by andreamari — COMMENTED** [review]
Thanks! 
I made 2 inline suggestions here on GihHub and they got formatted in a very strange way. Probably it is better to manually implement them instead of committing. They are very simple:

**Suggestion 1 (optional)**:  Move the line `The Unitary Fund research <a href="https://github.com/unitaryfund/research">GitHub repo</a>...` below the list of papers.

**Suggestion 2**: Add this line to the staff papers: `<li class="leading-block"><a href="https://arxiv.org/abs/2008.06517">Estimating the gradient and higher-order derivatives on quantum hardware</a></li><br>`

File: research.html (hunk 2 above)
Comment (crazy4pi314) [reply research.html:102]: Yeah good point. I guess my logic was "variety of projects" didn't mean all of them and having it first increased the likelihood they would click on it. Let me try tweaking the wording on this so having it first sets expectations accordingly.

**Comment by crazy4pi314** [comment]
Thanks @andreamari! I have address your comments, and I think the main thing we need now is the microgrant generated papers. Is there a place we are tracking those @nathanshammah?

**Review by nathanshammah — APPROVED** [review]
LGTM, thank you @crazy4pi314!

**Pull request merged**